docs/ENV.md

```diff
@@ -99,3 +99,19 @@ Shortcode for the default tenant (e.g. plio)
 
 #### `DEFAULT_TENANT_DOMAIN`
 The domain for the default tenant (e.g. 0.0.0.0 locally, plio.in on production)
+
+
+### Auth0 for Plio Analytics
+While setting up Plio analytics, you need to make sure the following variables are also updated. These are responsible to fetch an access token from Auth0 Identity Provider.
```

Comment:
I am guessing we'll have to make another sub-domain for the production one right? 
Also, is any sort of payment set up for this?